torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5654,6 +5654,16 @@ def sample_inputs_pairwise_distance(op_info, device, dtype, requires_grad, **kwa
         SampleInput(make(shape), args=(make(shape),), kwargs=kwargs) for shape, kwargs in shapes_and_kwargs
     ]
 
+def sample_inputs_pixel_shuffle(op_info, device, dtype, requires_grad, **kwargs):
+    return [
+        SampleInput(
+            make_tensor((1, 9, 2, 2), device=device, dtype=dtype, requires_grad=requires_grad),
+            kwargs=dict(upscale_factor=upscale_factor),
+        )
+        for upscale_factor in (1, 3)
```

Comment:
The docs for pixel_shuffle https://pytorch.org/docs/stable/generated/torch.nn.PixelShuffle.html suggest that it accepts tensors that aren't 4D. Could we test something of size (9, 2, 2) and (2, 3, 9, 2, 2) ?